resources/public/api/conf/1.0/schemas/retrieve_se_bsas.json

```diff
@@ -0,0 +1,264 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Retrieve a Self-Employment Business Source Accounting Summary (BSAS)",
+  "description": "This endpoint allows the user to request a specific Business Source Accounting Summary using a unique identifier. By default, when it is available the adjusted summary will be returned. A filter can be set to specify whether the original or the adjusted is to be returned.",
+  "type": "object",
+  "required": ["metadata"],
+  "properties": {
+    "metadata": {
+      "description": "Object containing the identifying data for the business and associated information for the BSAS with this ID",
+      "type": "object",
+      "required": [
+        "typeOfBusiness",
+        "selfEmploymentId",
+        "accountingPeriod",
+        "requestedDateTime",
+        "bsasId",
+        "summaryStatus",
+        "adjustedSummary"
+      ],
+      "properties": {
+        "typeOfBusiness": { "$ref": "definitions.json#/definitions/typeOfBusiness"},
+        "selfEmploymentId": { "$ref": "definitions.json#/definitions/selfEmploymentId"},
+        "accountingPeriod": {"$ref": "definitions.json#/definitions/accountingPeriod" },
+        "taxYear": {"$ref": "definitions.json#/definitions/taxYear" },
```

Comment:
taxYear in the spec says `The tax year into which this accounting period falls` - are we okay with the documentation text being different?